psydac/fem/tensor.py

```diff
@@ -53,35 +53,21 @@ class TensorFemSpace( FemSpace ):
 
     """
 
-    def __init__( self, *args, **kwargs ):
+    def __init__( self, domain_h, *args, **kwargs ):
```

Comment:
done in [b9041f7](https://github.com/pyccel/psydac/pull/232/commits/b9041f73c9a1d5dc0b465b4ad51f6568ac73555d)

Reply:
You changed `domain_h` to `domain_decomposition`, but you did not change the signature of the constructor.

Reply:
ah yes, you are right, about the spaces should we keep the `*`  and have the signature like this
`def __init__(self, decomposition, *spaces, vector_space=None, cart=None):`?
because this changes the way we create it

Reply:
Yes, it is better to keep `*spaces` as you suggest

Reply:
Just be careful that the last two arguments become automatically keyword-only as they follow a starred argument.

Reply:
fixed in [c736d3e](https://github.com/pyccel/psydac/pull/232/commits/c736d3e4420e9a4750c049a8d09a558f908abfd9)